cves/kernel/CVE-2017-5548.yml

```diff
@@ -371,8 +392,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: False
+  note: | 
+    Doesn't seem like an order of operations, more of a problem with memory allocation 
```

Comment:
Many notes are written in a passive voice.  Consider removing phrases such as "Doesn't seem" to instill a more confident tone.